### DISCUSSION TITLE
betterrecycling3

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#15] [Announcements]
# betterrecycling3

Exploring solutions to plastic recycling, food waste, container return schemes

https://nonzerosum.games/betterrecycling3.html

**Comment by andrewtaneglen** [comment]
I'd favour the versatility of aluminium. I think it requires more energy to work with, but energy use should not be considered a long term sustainability issue. Imagine two recycle bins, one for glass, one for aluminium, with both actually being recycled.

**Reply by jamesstephenbrown** [reply]
Absolutely, in my research aluminium got pretty much top marks all round as a recycling success story - especially when you can make clothes out of it and all that (I always find that so baffling when something silky soft is made out of aluminium) I wonder if they could solve the "squeeziness" issue with aluminium? If they can make a jacket, why not a flexible container?
